build-release.sh

```diff
@@ -152,10 +152,11 @@ if [ "${build_classical}" == "1" ]; then
     osx_tmpdir=$(ssh "${OSX_HOST}" "mktemp -d")
     
     scp "${reldir}/${binname}.zip" "${OSX_HOST}:${osx_tmpdir}"
+    scp "${basedir}/build-macosx/editor.entitlements" "${OSX_HOST}:${osx_tmpdir}"
```

Comment:
Not really sure if `${basedir}/build-macosx` is the correct path to use.

Reply:
Should be fine as long as its accessible by code sign, the entitlements file itself does not end up in your final bundle. It's just data that is signed along with your app.

Reply:
that said I think xcode just places it in the root of your project folder.

Reply:
> Should be fine as long as its accessible by code sign, the entitlements file itself does not end up in your final bundle. It's just data that is signed along with your app.

Well, it's copying `entitlements` file to remote macOS machine to make them accessible for code sign of the editor binary. The path it copies them from is the question really, since I'm not sure how script is used exactly. 
Maybe @hpvb can clarify.

Reply:
Yeah the path seems good.